### ISSUE TITLE
Suggestion: Add cinematic border to cutscenes

### ISSUE THREAD

**Issue by petes117** [KozGit/DOOM-3-BFG-VR#155]
Hi guys great mod, love playing it so far!

Just a quick idea/suggestion... wondering if it's possible for you guys to add the cinematic borders to the cutscenes in same way as from the codes4fun RBDOOM-3-BFG 1.1.0 mod (it was added in Nov 2016), I thought it worked really well.

Thanks, keep up the great work!

**Comment by tsr22** [comment]
Agreed! It would be cool to have it is an option4. It worked real well in RBDoom.
